server/routes/templates/infoForCreatedSuperadmin.hjs

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
It does not look like other mail templates.
In sendMailAdminInfo you have properties like initialGreeting, finalGreeting, where is that here?
We need to have the same static looks(template) for all messages.